include/swift/RemoteInspection/TypeRefBuilder.h

```diff
@@ -1871,7 +1871,7 @@ class TypeRefBuilder {
         if (auto symbol = OpaquePointerReader(
                 remote::RemoteAddress(adjustedProtocolDescriptorTarget),
                 PointerSize)) {
-          if (!symbol->getSymbol().empty()) {
+          if (!symbol->getSymbol().empty() && symbol->getOffset() == 0) {
```

Comment:
Why must the offset be 0 in this case?

Reply:
I just took this for granted without further investigation, but given that the Offset is never taken here, it seems plausible.